### ISSUE TITLE
rubocop config is outdated

### ISSUE THREAD

**Issue by bartes** [xinminlabs/rubocop-config-prettier#1]
Error: The `Layout/FirstParameterIndentation` cop has been renamed to `Layout/IndentFirstArgument`.
(obsolete configuration found in rubocop-config-prettier-0.1.0/config/rubocop.yml, please update it)
The `Layout/IndentArray` cop has been renamed to `Layout/IndentFirstArrayElement`.
(obsolete configuration found in rubocop-config-prettier-0.1.0/config/rubocop.yml, please update it)
The `Layout/IndentHash` cop has been renamed to `Layout/IndentFirstHashElement`.
(obsolete configuration found in rubocop-config-prettier-0.1.0/config/rubocop.yml, please update it)

**Comment by flyerhzm** [comment]
@bartes fixed and released version 0.1.1

**Comment by buckmaxwell** [comment]
I am getting this 

```
➜  casemanager git:(rufo) ✗ bundle exec rubocop
/Users/mbuck/.rvm/gems/ruby-2.6.3/gems/rubocop-config-prettier-0.1.11/config/rubocop.yml: Layout/LineLength has the wrong namespace - should be Metrics
Error: The `Layout/FirstParameterIndentation` cop has been renamed to `Layout/IndentFirstArgument`.
(obsolete configuration found in /Users/mbuck/.rvm/gems/ruby-2.6.3/gems/rubocop-config-prettier-0.1.11/config/rubocop.yml, please update it)
```

Looks like my v is 1.1.11 :/

**Comment by flyerhzm** [comment]
@buckmaxwell it's fixed in version 0.1.12
